server/templates/includes/nav.pug

```diff
@@ -1,6 +1,7 @@
 nav.primary
   .container
-    a.brand.is-monospace(href="/") #{sitename}
+    a.brand.is-monospace(href="/")
+      img(src="/static/brand.svg", alt=`${sitename}`)
```

Comment:
I kind of like that before you didn't *have* to set a brand image, could it be `CUSTOM_BRAND_URL` and then a little if statement here?

Reply:
You wouldn't need `brand.svg` too as well then?